README.md

```diff
@@ -1,46 +1,74 @@
-## TODO
+# Supaapps Auth Guard <!-- omit in toc -->
 
+- [Installation](#installation)
+  - [ENV vars](#env-vars)
+  - [Add new custom guard](#add-new-custom-guard)
+- [Usage example](#usage-example)
+- [Testing](#testing)
 
+## Installation
 
 `composer require supaapps/supaapps-guard`
 
+### ENV vars
 
-env vars:
+add env vars to your `.evn`:
 
-- `SUPAAPPS_GUARD_AUTH_SERVER_URL`
-- `SUPAAPPS_GUARD_AUTH_REALM`
-
-
----
+```env
+`SUPAAPPS_GUARD_AUTH_SERVER_URL=http://example.com`
+`SUPAAPPS_GUARD_AUTH_REALM_NAME=myapp`
+```
 
+### Add new custom guard
 
-### auth.php
+On `config/auth.php` add following lines
 
 ```php
-
-    'guards' => [
-        'jwt' => [
-            'driver' => 'supaapps-guard',
-            'provider' => 'users',
-        ],
+'guards' => [
+    'jwt' => [
+        'driver' => 'supaapps-guard',
+        'provider' => 'users',
     ],
-
+],
 ```
 
+## Usage example
 
-### example route
+on `routes/api.php`, add following lines
 
 ```php
-
 Route::middleware('auth:jwt')->get('/user', function (Request $request) {
     return [
         $request->user(),
-        auth()->firstName(),
-        auth()->lastName(),
-        auth()->email(),
-        auth()->scopes(),
-        auth()->scopesArray(),
-        ];
+        auth('jwt')->firstName(),
```

Comment:
with an assumption that by default APIs will have one driver `auth()` should be valid unless someone installs multiple drivers they know they have multiple drivers. or let's keep both mentioned

Reply:
We can mention in docs to set the default `gaurd` to `jwt`

```diff
'defaults' => [
        +'guard' => 'jwt',
        'passwords' => 'users',
],
```

Reply:
let's add it as a comment, but assume if someone is installing package it's on new project as default, then add a note if it's not default how to use